subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/PermissionChecker.kt

```diff
@@ -0,0 +1,73 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.app.UiAutomation
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.ParcelFileDescriptor
+import androidx.core.content.ContextCompat
+import java.io.FileInputStream
+import java.io.IOException
+import java.io.InputStream
+import java.util.*
+
+
+class PermissionChecker(private val context: Context) {
+
```

Comment:
Why don't we use it instead `com.avito.android.test.Device.grantPermissions`?
It does the same

Reply:
Ye, didnt know about Device.grantPermissions, replaced